prov/efa/src/efa_strerror.c

```diff
@@ -101,6 +102,21 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
+	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
+		return "This error can be caused by a crashed peer. "
+			   "Please verify the receiver status";
```

Comment:
Can we print peer address here?

Reply:
I think this function/file should be limited to returning `const` string fragments which callers can stitch together to build useful error messages. The peer address is more of a dynamic/runtime thing and should be a separate utility.

What we should do is defer all error reporting to central utilities which take the current state as input and log error messages (kind of like our `_write_error()` functions). That would give us more flexibility and allow us to easily configure different combinations of various information, rather than shoehorning as much information as possible in the error strings themselves.